server/apiserver/pool.go

```diff
@@ -6,15 +6,15 @@ type Pool struct {
 	Register   chan *Client
 	Unregister chan *Client
 	Clients    map[*Client]bool
-	Broadcast  chan Message
+	Broadcast  chan ChangeClientPixelMsg
 }
 
 func NewPool() *Pool {
 	return &Pool{
 		Register:   make(chan *Client),
 		Unregister: make(chan *Client),
 		Clients:    make(map[*Client]bool),
-		Broadcast:  make(chan Message),
+		Broadcast:  make(chan ChangeClientPixelMsg),
```

Comment:
ok this is under the assumption that we will only need to broadcast ChangeClientPixelMsg's through all the clients because I couldn't figure out polymorphism in Go to have all messages implement Msg